### PR TITLE
Api get all querys hash orderd

### PR DESCRIPTION
**Pull request by spetzreborn**
When calling api.php with key getAllQuerys, optionally set getAllQuerysto an value of:
    orderByClientDomainTime
    orderByClientTimeDomain
    orderByTimeClientDomain
    orderByTimeDomainClient
    orderByDomainClientTime
    orderByDomainTimeClient
    ANYTHING ELSE

If argument is one of orderBy* make getAllQueries() return with JSON
hash that is structured by the way the argument sets.

This make dataparsing from the api mush easier.

This does not break the originally function, and is backwards compatible.

### CODE REVIEW

**Comment by DL6ER** [comment]
I see that solving codacy's issue would make the code even less readable than it is right now (we would have to factor things out into separate functions).

@dschaper We should tell codacy to ignore this "issue".

**Comment by spetzreborn** [comment]
Also, function getRecentItems($qty) a few functions higher in the same file, data.php uses this type of declaration.

File: README.md [hunk 1]
```diff
@@ -23,3 +23,6 @@ A read-only API can be accessed at `/admin/api.php`. With either no parameters o
 ```
 
 There are many more parameters, such as `summaryRaw`, `overTimeData`, `topItems`, `recentItems`, `getQueryTypes`, `getForwardDestinations`, `getQuerySources`, and finally `getAllQueries`.
+
+`getAllQueries` can optionally be set with one this values to return JSON hash ordered as value implies: `orderByClientDomainTime`. `orderByClientTimeDomain`, `orderByTimeClientDomain`, `orderByTimeDomainClient`, `orderByDomainClientTime` or `orderByDomainTimeClient`.
```
Comment (Mcat12) [inline README.md]: `one of these values`

**Comment by Mcat12** [comment]
Chrome's network tools shows this functionality off pretty well:
Image

**Comment by Mcat12** [comment]
Approved

**Comment by DL6ER** [comment]
Yes, that looks fine.

Approved

**Pull request merged**